uber/site_sections/preregistration.py

```diff
@@ -260,7 +260,7 @@ def dealer_confirmation(self, session, id):
     @log_pageview
     def group_members(self, session, id, message=''):
         group = session.group(id)
-        charge = Charge([group, group.leader]) if group.leader else Charge(group)
+        charge = Charge([group] + listify(group.attendees))
```

Comment:
OK, so the original intention of this was to actually charge people for attendees' unpaid kick-in levels. This `charge` value gets passed to the stripe form on the page itself -- the other part of the code is what we mark the attendees using. Getting rid of this would normally have made us mark attendees as paid but not actually charge for their unpaid kick-in levels.

However, since this PR was submitted, I changed a lot about how group payments work. I'm about to update the branch with the new code and I'll explain it in full.